src/main/java/org/broadinstitute/hellbender/tools/walkers/CembaTypeCaller.java

```diff
@@ -0,0 +1,146 @@
+                    .map(SAMReadGroupRecord::getSample)
+                    .sorted()
+                    .distinct()
+                    .collect(Collectors.toList());
+
+            return new VCFHeader(headerLines, samples);
+        }
+
+        return null;
+    }
+
+
+    @Override
+    public void apply(AlignmentContext alignmentContext, ReferenceContext referenceContext, FeatureContext featureContext) {
+        if(referenceContext.hasBackingDataSource()) {
+            String strand = new String(referenceContext.getBases());
+            int unconverted_bases, converted_bases = unconverted_bases = 0;
+            String alt, context = alt = null;
+
+            // check the forward strand for methylated coverage
+            if(strand.equals("C")) {
+                ReadPileup forwardBasePileup = alignmentContext.stratify(AlignmentContext.ReadOrientation.FORWARD).getBasePileup();
+                // unconverted: C; converted: T
+                unconverted_bases = forwardBasePileup.getBaseCounts()[1];
+                converted_bases = forwardBasePileup.getBaseCounts()[3];
+
+                // if there is methylated coverage
+                if (unconverted_bases + converted_bases > 0) {
+                    alt = "T";
+                    context = new String(referenceContext.getBases(0,2));
+                }
+            }
+            // check the reverse strand for methylated coverage
+            else if(strand.equals("G")) {
+                ReadPileup reverseBasePileup = alignmentContext.stratify(AlignmentContext.ReadOrientation.REVERSE).getBasePileup();
+                // unconverted: G; converted: A
+                unconverted_bases = reverseBasePileup.getBaseCounts()[2];
+                converted_bases = reverseBasePileup.getBaseCounts()[0];
+
+                // if there is methylated coverage
+                if (unconverted_bases + converted_bases > 0) {
+                    alt = "A";
+                    // get the reverse complement for context b/c we are on the reverse strand
+                    context = new String(org.broadinstitute.hellbender.utils.BaseUtils.simpleReverseComplement(
+                            referenceContext.getBases(2,0)));
+                }
+            }
+
+            // if there are reads that have methylated coverage
+            if (alt != null) {
+                LinkedHashSet<Allele> alleles = new LinkedHashSet<>();
+                alleles.add(Allele.create(strand, true));
+                alleles.add(Allele.create(alt, false));
+
+                VariantContextBuilder vcb = new VariantContextBuilder();
+                vcb.chr(alignmentContext.getContig());
+                vcb.start(alignmentContext.getPosition());
+                vcb.stop(alignmentContext.getPosition());
+                vcb.noID();
+                vcb.alleles(alleles);
+                vcb.noGenotypes();
+                vcb.unfiltered();
+                vcb.attribute("mC", unconverted_bases);
```

Comment:
It might make sense to write these as 1 tag with 2 values (you could use the already existing `AD` field which is allele depth.) Instead of as 2 separate tags.

Reply:
I left them as 2 tags in order to maintain consistency with the ALLC files